mozci/console/commands/push.py

```diff
@@ -226,26 +228,30 @@ def handle(self) -> None:
 
             # Send a notification when some emails are declared in the config
             emails = config.get("emails", {}).get("classifications")
-            if emails:
+            matrix_room = config.get("matrix-room-id", None)
+            if emails or matrix_room:
                 # Load previous classification from taskcluster
                 try:
                     previous = push.get_existing_classification()
                 except SourcesNotFound:
                     # We still want to send a notification if the current one is bad
                     previous = None
 
-                self.send_emails(emails, push, previous, classification, regressions)
+                self.send_notifications(
+                    emails, matrix_room, push, previous, classification, regressions
+                )
 
-    def send_emails(
+    def send_notifications(
```

Comment:
Tests will need to be adjusted following this change

Reply:
Oups I forgot that there were any